languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -54706,15 +54708,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example>Tu envisages de venir ?</example>
                 <example>Actes de naissance</example>
                 <example>Tu me manques</example>
+                <example>Tu n'y arrives pas</example>
                 <example>Des choses brillantes</example>
+                <example>Tu ne nous le montres pas</example>
+                <example>Toi et moi sommes perdus</example>
             </rule>
             <!-- Règle pour les Erreurs d'Orthographe Communes cameras|tractes|magasines|députes|délègues|cosignes|arrives|jures|immigres|troues|lutes -->
             <rule>
                 <pattern>
                     <!-- Anything plural, except a verb -->
                     <token postag="[^V].* p$" postag_regexp="yes"/>
                     <marker>
-                        <token regexp="yes">cameras</token>
+                        <token>cameras</token>
                     </marker>
                 </pattern>
                 <message>Cameras est l'indicatif de "se camer". Vouliez-vous dire <suggestion>caméras</suggestion> ?</message>
```

Comment:
**Formatting:** The target string should also have quotes.I know sometimes it's missing in the existing rules but most of them have this pattern: 
`
            <message>L'usage de la préposition « sur » après le verbe « primer » est déconseillé, vous pouvez aussi choisir la forme « l'emporter sur ».</message>`